src/dlgIRC.cpp

```diff
@@ -45,7 +45,12 @@ QString dlgIRC::DefaultNickName = QStringLiteral("Mudlet");
 QStringList dlgIRC::DefaultChannels = QStringList() << QStringLiteral("#mudlet");
 int dlgIRC::DefaultMessageBufferLimit = 5000;
 
-dlgIRC::dlgIRC(Host* pHost) : mReadyForSending(false), mpHost(pHost), mIrcStarted(false), mInputHistoryMax(8), mConnectedHostName()
+dlgIRC::dlgIRC(Host* pHost) 
+: mReadyForSending(false)
+, mpHost(pHost)
+, mIrcStarted(false)
+, mInputHistoryMax(8)
+, mConnectedHostName()
```

Comment:
```suggestion
:mReadyForSending(false)
,mpHost(pHost)
,mIrcStarted(false)
,mInputHistoryMax(8)
,mConnectedHostName()
```

Reply:
:-1:  We do space out the initialisation list like @itsTheFae has it.

Reply:
So we do!